mica/starcheck/starcheck_parser.py

```diff
@@ -107,7 +107,7 @@ def get_dither(obs_text):
         raise ValueError("No OBSID found for this catalog")
     for oline in targ_search.group(1).split("\n"):
         dither_re = re.match(
-            "Dither:\s(\S+)\s+Y_amp=\s*(\S+)\s+Z_amp=\s*(\S+)\s+Y_period=\s*(\S+)\s+Z_period=\s*(\S+)\s*",
+            "Dither:\s+(\S+)\s+Y_amp=\s*(\S+)\s+Z_amp=\s*(\S+)\s+Y_period=\s*(\S+)\s+Z_period=\s*(\S+)\s*",
```

Comment:
I'll try to just get in a whitespace fix in starcheck so we can skip the change, but wanted this ready.